src/main/java/edu/kis/powp/jobs2d/features/CommandParser.java

```diff
@@ -0,0 +1,97 @@
+package edu.kis.powp.jobs2d.features;
```

Comment:
Wrong location of the class. Please use commands package.

Reply:
Moved to command package